pkg/gits/github.go

```diff
@@ -791,25 +791,51 @@ func (p *GitHubProvider) ListOpenPullRequests(owner string, repo string) ([]*Git
 	return answer, nil
 }
 
+func extractRepositoryCommitAuthor(rc *github.RepositoryCommit) (gu *GitUser) {
+	commitAuthorPresent := rc.Commit.Author != nil
```

Comment:
For what it's worth, technically I think there will always be a `rc.Commit.Author`, since git does not let you commit anything without author and email set. So in my world, we can just create the GitUser with this name and email and then just amend any additional information like URL and login in case the `rc.Author` is not nil which basically means that GitHub was able match the commit author against a GitHub user (which might not need to be the case). Does that make sense?

Reply:
yup I think it's clear, let me do the changes and I'll request your feedback again